build_bin/kairos_env_init

```diff
@@ -95,6 +95,14 @@ else
   output "Found wired virtualenv"
 fi
 
+# force processing of dependency links
```

Comment:
Consider making this a file that can be edited and have the script that build virtualenvs copy - makes it easier to edit and understand changes in the long run.

Reply:
My last build called this code multiple times, for reasons unclear to me.  I'm migrating it into wire_virtualenv because that automatically checks that it doesn't get re-run if the pip.conf already exists, and just putting it into the block of stuff that's already getting catted into the pip.conf.  Will see if that fixes it, if so, then this is moot.